packages/client/src/services/useAuth/index.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+import { getUserInfo } from '../../api/basic/auth'
+
+export function useAuth() {
+  const [isAuth, setIsAuth] = useState<boolean>(false)
+  const [isAuthLoading, setIsAuthLoading] = useState<boolean>(true)
+  const [error, setError] = useState<Error | null>(null)
+
+  useEffect(() => {
```

Comment:
you should add additional useAsyncEffect hook (find please how implement this) and use getUserInfo with async await